app/src/organisms/OnDeviceDisplay/RobotDashboard/RecentRunProtocolCard.tsx

```diff
@@ -144,13 +158,22 @@ export function ProtocolWithLastRun({
       borderRadius={BORDERS.borderRadiusSize4}
       onClick={handleCardClick}
     >
-      <Flex>
+      <Flex justifyContent={JUSTIFY_SPACE_BETWEEN}>
         <Chip
           paddingLeft="0"
           type={isReadyToBeReRun ? 'success' : 'warning'}
           background={false}
           text={i18n.format(chipText, 'capitalize')}
         />
+        {showSpinner && (
+          <Icon
+            name="ot-spinner"
+            aria-label="icon_ot-spinner"
+            spin={true}
+            size="2.5rem"
+            color={COLORS.darkBlackEnabled}
```

Comment:
darkBlackEnabled and darkBlack100 are the same but this component is for ODD so let's use darkBlack100

```suggestion
            color={COLORS.darkBlack100}
```